assemblies/assembly-bulk-importing-from-github.adoc

```diff
@@ -0,0 +1,15 @@
+= Bulk Import GitHub repositories
+
+Bulk Import is a {company-name} *Tech Preview* feature.
```

Comment:
```suggestion
Bulk Import is a https://access.redhat.com/support/offerings/techpreview[{company-name} *Tech Preview*] feature.
```

I think it might help to link to the Support pages for a better understanding of the support scope.